### PR TITLE
refactor(facade): refactor type check function - is*()

### PR DESCRIPTION
**Pull request by vicb**
… ListWrapper

### CODE REVIEW

**Comment by vicb** [comment]
@mprobst if you agree with the changes, I'll move `PromiseWrapper.isPromise()` to the lang facade as well.

File: modules/angular2/src/facade/lang.ts [hunk 1]
```diff
@@ -79,6 +79,10 @@ export function isMap(obj): boolean {
   return typeof obj === 'object' && obj !== null;
 }
 
+export function isList(obj): boolean {
```
Comment (mprobst) [inline modules/angular2/src/facade/lang.ts]: If I understood @mhevery correctly, the plan is to harmonize on JavaScript terminology in our code base, i.e. use `isArray`, not `isList`, for consistency. Does that make sense?

**Comment by mprobst** [comment]
See comment, I think we should go for `isArray`, and at that point it might be easier to just write `Array.isArray` directly.

**Comment by vicb** [comment]
@mprobst PTAL

**Comment by mprobst** [comment]
LGTM.

**Comment by vicb** [comment]
Thanks @mprobst

**Comment by vicb** [comment]
landed as cdfb635737a939bfdde495071686f11023b3f52e

**Pull request closed without merging**